requirements.txt

```diff
@@ -1,4 +1,4 @@
-# wxpython
+wxpython==3.0.0.0
```

Comment:
This version works for me and it is available in the default channel. So let's go with this one for now.